packages/block-library/src/table/theme.scss

```diff
@@ -5,10 +5,6 @@
 		border-bottom: 3px solid;
```

Comment:
We should remove the `thead` border styles here as well, as they've been moved to style.scss.

Reply:
Hello @mikachan 

Okay, I will make this change and will add another PR.

Thanks for your feedback.

Reply:
Hello @mikachan 

I have made above mentioned change and removed extra css from theme.scss file.

Please let me know your feedback or if you will find anything.

Thanks,